### PR TITLE
#1339 Feature/form99 submit

### PR DESCRIPTION
**Pull request by toddlees**

### CODE REVIEW

File: front-end/src/app/reports/f99/f99-routing.module.ts [hunk 1]
```diff
@@ -34,7 +37,45 @@ const routes: Routes = [
     title: 'Print preview',
     component: PrintPreviewComponent,
     resolve: { report: ReportResolver, sidebar: SidebarStateResolver },
-    data: { sidebarSection: ReportSidebarSection.REVIEW },
+    data: {
+      sidebarSection: ReportSidebarSection.REVIEW,
+      getBackUrl: (report?: Report) => '/reports/f99/edit/' + report?.id,
+      getContinueUrl: (report?: Report) => '/reports/f99/submit/step1/' + report?.id,
```
Comment (mjtravers) [inline front-end/src/app/reports/f99/f99-routing.module.ts:43]: Good idea to put this in the router.

File: front-end/src/app/reports/submission-workflow/submit-report-step2.component.html [hunk 2]
```diff
@@ -83,20 +85,20 @@ <h3>Enter password</h3>
...
           </div>
         </div>
         <div class="col-6">
           <div class="field">
-            <label for="filing_password">
+            <label for="filingPassword">
               <strong>ELECTRONIC FILING PASSWORD </strong>
             </label>
-            <input type="text" pInputText id="filing_password" formControlName="filing_password"/>
+            <input type="text" pInputText id="filingPassword" formControlName="filingPassword" />
```
Comment (mjtravers) [inline front-end/src/app/reports/submission-workflow/submit-report-step2.component.html:98]: Let's make this type="password" to hide it when being typed out.

**Review by mjtravers — APPROVED** [review]
Not part of scope for this ticket, but I think we should change the password input to type="password" to hide it while the user is typing it in.

File: front-end/src/app/reports/submission-workflow/submit-report-step2.component.html (hunk 2 above)
Comment (toddlees) [reply front-end/src/app/reports/submission-workflow/submit-report-step2.component.html:98]: side effects of setting it to type="password" moved this out of scope

**Pull request merged**